### DISCUSSION TITLE
Password / Protected Folders

### DISCUSSION THREAD

**Discussion by Struntzi** [retypeapp/retype#341] [Q&A]
Hey, 

is there any possibility to protect specific folders?
Either with a password or for a specific group of ip addresses or so? 

I would like to share some information with just some specific group of people, but where the whole left wiki should be free to use for all. 

Maybe anyone have a solution for this.

Greets

**Comment by danielcharrua** [comment]
I'm interested too on this feature

**Comment by geoffreymcgill** [comment]
We were just discussing this feature and had planned to include in the upcoming Retype `v3.0` release.

Retype should be able to provide custom password protection for individual pages and entire folders.

Retype would not be able to limit access by IP address, as that would be a server-side configuration. If your web server allows for `.htaccess` and `.htpasswd` file configuration, you should be able to use those to limit file or folder access. See https://help.dreamhost.com/hc/en-us/articles/216363187-Password-protecting-your-site-with-an-htaccess-file. 

If you are using GitHub Pages, to the best of my knowledge, you cannot configure an `.htaccess` or `.htpasswd` file.

If you are using [Cloudflare](https://www.cloudflare.com/) for your DNS hosting/configuration, I do believe they have some password protecting options. I have not implemented a password protected site using Cloudflare, but I have seen some features within the domain configuration that look like they should apply to this scenario.

Hope this helps.

I will keep this thread updated with our progress.

**Reply by danielcharrua** [reply]
@geoffreymcgill great!

Do you have an ETA for v3.0 release? Are we talking weeks, months?

Thank you.
